core/order_test.go

```diff
@@ -356,3 +358,27 @@ func TestOpenBazaarNode_CalculateOrderTotal(t *testing.T) {
 		t.Error("Calculated wrong order total")
 	}
 }
+
+func TestOpenBazaarNode_GetOrder(t *testing.T) {
+	node, err := test.NewNode()
+	if err != nil {
+		t.Error(err)
+	}
+
+	contract := factory.NewContract()
```

Comment:
We should manually set the state of the `contract` object instead of depending on the factory to never change (as it might). Please manually define the `Funded` and `State` fields which are checked below.

Reply:
done